src/components/templates/Search/filterService.module.css

```diff
@@ -47,10 +47,14 @@ button.filter,
   margin-bottom: calc(var(--spacer) / 6);
 }
 
+.showClear:hover{
+  display: inline-flex;
+  color: var(--color-primary) !important;
+}
 .showClear {
   display: inline-flex;
   text-transform: capitalize;
-  color: var(--color-secondary);
+  color: var(--color-secondary) !important;
```

Comment:
Are these `!important` declarations necessary? I think it should work without them

Reply:
True, in this case it works without them; just have tested.